src/web_interface/security/authentication.py

```diff
@@ -1,27 +1,35 @@
 import base64
 import os
 
-from flask_security import Security, UserMixin, RoleMixin, AnonymousUser
+from flask_security import AnonymousUser, LoginForm, RoleMixin, Security, UserMixin, uia_username_mapper
 from flask_sqlalchemy import SQLAlchemy
 from werkzeug.local import LocalProxy
+from wtforms import StringField
+from wtforms.validators import DataRequired
 
 from web_interface.security.privileges import PRIVILEGES
 from web_interface.security.user_role_db_interface import UserRoleDbInterface
 
 
-def add_flask_security_to_app(app, config):
-    _add_configuration_to_app(app, config)
+def add_config_from_configparser_to_app(app, config):
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['SECURITY_PASSWORD_SALT'] = config.get('data_storage', 'password_salt').encode()
+    app.config['SQLALCHEMY_DATABASE_URI'] = config.get('data_storage', 'user_database', fallback='sqlite:///')
+    # TODO fix redirect loop here
+    app.config['SECURITY_UNAUTHORIZED_VIEW'] = '/login'
+    app.config['LOGIN_DISABLED'] = not config.getboolean('ExpertSettings', 'authentication')
 
-    db = SQLAlchemy(app)
+    # As we want to use ONLY usernames and no emails but email is hardcoded in
+    # flask-security we change the validation mapper of 'email'.
+    app.config['SECURITY_USER_IDENTITY_ATTRIBUTES'] = [{'email': {"mapper": uia_username_mapper, "case_insensitive": True}}]
 
-    user_interface = create_user_interface(db)
-    security = Security(app, user_interface)
 
-    _add_apikey_handler(security, user_interface)
-    return db, user_interface
+def add_flask_security_to_app(app):
+    # db.Table etc cause errors but are correct
```

Comment:
I did not observe any errors. Should I see errors?

Reply:
For me pylint complains that `Table` is not a member of `db`.
Pylint is wrong so I disabled the warnings and added this comment to explain why I disabled them.